src/encoded/tests/data/inserts/target.json

```diff
@@ -377,71 +241,51 @@
     {
         "uuid": "14173a5f-799e-4dae-b155-a142c5b6b566",
         "label": "H3K9me3",
-        "gene_name": "HIST2H3C",
+        "genes": ["126961"],
         "organism": "human",
         "investigated_as": [
             "histone"
         ],
-        "status": "released",
-        "dbxref": [
-            "UniProtKB:Q71DI3"
-        ]
+        "status": "released"
     },
     {
         "uuid": "1aa5391a-01f1-4983-bdb6-d93db5f53334",
         "label": "H3K4me1",
-        "gene_name": "HIST2H3C",
+        "genes": ["126961"],
         "organism": "human",
         "investigated_as": [
             "histone"
         ],
+        "status": "released"
+    },
+    {
         "status": "released",
-        "dbxref": [
-            "UniProtKB:Q71DI3"
+        "uuid": "452c5ea6-5880-499d-b8bb-392790fd2b89",
+        "organism": "human",
+        "genes": [
+            "8335",
```

Comment:
This is because there are two identical copies of H2A right?

Reply:
Yes, and there are actually more. Please check my comments in WRAN-911. In my current plan, WRAN-911 needs to get in first before we can merge this.